registration/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+
+
+class Source(models.Model):
+    """ The source from which a registation originates.
+    """
+    name = models.CharField(max_length=100, null=False, blank=False)
+    created_at = models.DateTimeField(auto_now_add=True)
```

Comment:
Missing the FK to the user model so we can determine source from API key on POST request